website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -40,18 +40,18 @@ segmented below into two categories: required and optional parameters. Within
 each category, the available configuration keys are alphabetized.
 
 In addition to the options listed here, a
-[communicator](/docs/templates/communicator.html) can be configured for this
+[communicator](../templates/communicator.html.md) can be configured for this
```

Comment:
Big warning: I'm not sure if the build toolchain you're using will tolerate the change I made to the md links...

Ideally it should, i.e., at build time it should convert this to `../templates/communicator.html`. If it doesn't, then you really don't want that part of this change. (And I should instead file a bug asking it to support that.)

Reply:
Yeah, our website doesn't currently support this change.

Reply:
Filed #6882.

Do you know if the generator at least supports `../` instread of `/docs` ?

I can just drop the change entirely and wait for the issue to be fixed, but at the very least, using relative paths gives a hint to everyone about where files are...

Reply:
Unfortunately I'm less familiar with that aspect. I know we use middleman through this gem https://github.com/hashicorp/middleman-hashicorp which might have functionality to do that.

To clarify, if you can come up with a way to get middleman to generate correct html while fixing the links in our markdown, I'm also open to that idea (however the partials will still be a problem)